### PR TITLE
Feat/component/footer

### PR DESCRIPTION
**Pull request by harrisazmi**
Component Footer

### CODE REVIEW

File: packages/react/src/components/footer.tsx [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+  FooterContentLinkSectionWrapper,
+  FooterContentLinkSectionTitle,
+  FooterContentLinkSectionContent,
+  FooterGeneralLink,
+  FooterDisclaimer,
+  FooterDisclaimerGroup,
+  FooterDisclaimerGroupRights,
+  FooterDisclaimerGroupDisclaimer,
+  FooterDisclaimerLastUpdate,
+};
```
Comment (irfancoder) [inline packages/react/src/components/footer.tsx:279]: Too many exports. I think certain items can be taken from other component OR does not require it's own component. 

Eg. 
1. `FooterAgencyMediaLink` => `Link`
2. `FooterGeneralLink` => `Link`
3. `FooterDisclaimerGroupRights` 
4. `FooterDisclaimerGroupDIsclaimer` 
5. `FooterDisclaimerLastUpdate` 
6. etc

File: packages/react/src/icons/facebook.tsx
Comment (irfancoder) [inline packages/react/src/icons/facebook.tsx:1]: Need to add original svg under `src/svg/icons` & run the `pnpm turbo gen icon`

**Review by irfancoder — CHANGES_REQUESTED** [review]

File: packages/react/src/components/footer.tsx [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+  className?: string;
+};
+type FooterAgencyMediaLinksLogoPathProps = {
+  children?: React.ReactNode;
+  className?: string;
+  icons: {
+    href: string;
+    icon: React.ReactNode;
+  }[];
+};
```
[the remark this answers is not in the document]
Comment (harrisazmi) [reply packages/react/src/components/footer.tsx:24]: ok boss, roger.

File: packages/react/src/components/footer.tsx (hunk 1 above)
Comment (harrisazmi) [reply packages/react/src/components/footer.tsx:279]: ok boss, i try reducing it.

File: packages/react/src/icons/facebook.tsx
Comment (harrisazmi) [reply packages/react/src/icons/facebook.tsx:1]: ok boss, roger.

**Comment by harrisazmi** [comment]
Footer Content (Open Up) ✅ : Rework as Last implementation was too rigid
Social Media Content (Open Up) ✅ : Reowork as Last implementation was too rigid
Time (Use ISO) ✅ 
Naming Rework Following new reworked PR ✅
Fix Hover issue ✅

**Review by irfancoder — APPROVED** [review]

**Pull request merged**